app/services/database_operations/database_operations.py

```diff
@@ -1,50 +1,55 @@
-from app import bcrypt
-from models import database
-from app.models import (Admins, RobotProfile, User, UserCriteria,
-                             UserRobot, engine, sessionmaker, Profile)
-from sqlalchemy import exists
 from geoalchemy2.elements import WKTElement
-from app.services.geolocalization_services.user_localization_and_distance import get_coordinates
-Session = sessionmaker(bind=engine)
-
-
-def user_in_database(password, username):
-    session = Session()
+from geoalchemy2.functions import ST_DWithin
+from sqlalchemy import and_, exists, not_
+from sqlalchemy.orm import joinedload
 
-    user_exists = session.query(exists().where(User.username == username)).scalar()
+from app import bcrypt
+from app.models import (
+    Matches,
+    Profile,
+    RobotProfile,
+    User,
+    UserCriteria,
+    UserRobot,
+    engine,
+    sessionmaker,
+)
+from app.services.geolocalization_services.user_localization_and_distance import (
+    get_coordinates,
+)
 
-    if user_exists:
-        user = session.query(User).filter_by(username=username).first()
-        print(user.password)
-        print(password)
-        if bcrypt.check_password_hash(user.password, password):
-            session.close()
-            return True
+Session = sessionmaker(bind=engine)
+session = Session()
 
-    session.close()
-    return False
 
+class UserObject:

```

Comment:
For me it's not a good choice for using  a class, these functionalities you can do with normal functions

Reply:
Yes but in class we can initialize one function once like in this file not multiple times like in functions.